packages/pwa-buildpack/lib/Utilities/serve.js

```diff
@@ -29,12 +30,23 @@ module.exports = async function serve(dirname) {
                     ...config.section('imageOptimizing'),
                     ...config.section('imageService')
                 });
+                if (process.env.ENABLE_EXPRESS_SERVER_COMPRESSION === 'true') {
+                    app.use(
+                        compression({
+                            threshold: 0
+                        })
+                    );
+                }
             }
         }
     );
 
     let envPort;
-    if (process.env.PORT) {
```

Comment:
This is a false positive all along. Since we dint have tests, we never found it. I have added covering tests for this change.